cmd/openshift-install/create.go

```diff
@@ -111,15 +111,16 @@ var (
 				}
 
 				timer.StartTimer("Bootstrap Complete")
-				err = waitForBootstrapComplete(ctx, config)
-				if err != nil {
+				if errInfo := waitForBootstrapComplete(ctx, config); errInfo != nil {
```

Comment:
Use `err` instead of `errInfo`. It is OK to hide the `err` declared in the outer scope. I don't know what `errInfo` means.